packages/ripple-nuxt-tide/lib/middleware/page-head.js

```diff
@@ -17,10 +17,14 @@ const tidePageHead = (context, pageData) => {
     // Set image.
     const featuredImage = pageData.tidePage.field_featured_image ? pageData.tidePage.field_featured_image.field_media_image : null
     const sectionImage = siteSection && siteSection.field_site_og_image ? siteSection.field_site_og_image.field_media_image : null
+    const twitterImage = siteSection && siteSection.field_site_twitter_image ? siteSection.field_site_twitter_image.field_media_image : null
     const primaryImage = pageData.tidePage.field_node_primary_site && pageData.tidePage.field_node_primary_site.field_site_og_image ? pageData.tidePage.field_node_primary_site.field_site_og_image.field_media_image : null
     const mediaImage = (featuredImage || sectionImage || primaryImage || null)
+    const mediaTwitterImage = (twitterImage || mediaImage || null)
     const image = mediaImage ? mediaImage.url : `${context.store.state.tide.protocol + '//' + context.store.state.tide.host}/img/social-media-image.jpg`
     const imageAlt = mediaImage ? mediaImage.meta.alt : ''
+    const imageTwitter = mediaTwitterImage ? mediaTwitterImage.url : `${context.store.state.tide.protocol + '//' + context.store.state.tide.host}/img/social-media-image.jpg`
+    const imageTwitterAlt = mediaTwitterImage ? mediaTwitterImage.meta.alt : ''
```

Comment:
Hey @kurtfoster just curious, any reason Twitter need a dedicated image instead of the existing `mediaImage`?

Reply:
@tim-yao , twitter and FB use different aspect ratios by default. The PPO asked for the change, because their square logo is perfect on Twitter, but is too large in FB. You can see details here, https://digital-engagement.atlassian.net/browse/SDPSUP-1959 and here, https://iamturns.com/open-graph-image-size/

Reply:
@kurtfoster Thanks, it make sense!